src/package_io/parse_input.star

```diff
@@ -295,15 +294,14 @@ def parse_network_params(input_args):
     ):
         fail("electra can only happen with capella genesis not bellatrix")
 
-    required_num_validators = 2 * result["network_params"]["slots_per_epoch"]
     actual_num_validators = (
         total_participant_count
         * result["network_params"]["num_validator_keys_per_node"]
     )
-    if required_num_validators > actual_num_validators:
+    if MIN_VALIDATORS > actual_num_validators:
         fail(
-            "required_num_validators - {0} is greater than actual_num_validators - {1}".format(
-                required_num_validators, actual_num_validators
+            "MIN_VALIDATORS - {0} is greater than actual_num_validators - {1}".format(
```

Comment:
can we do something like

"We require at least {0} validators but got {1}", MIN_VALIDATORS, actual_num_validators"